pyproject.toml

```diff
@@ -63,6 +63,8 @@ mypy = "^0.931"                 # Type checking
 click = "^8.0.3"                # Framework for building cli's
 ipdb = "^0.13.9"                # ipython breakpoints
 black = "^21.12b0"              # Code formatting and linting (with pyproject.toml support)
+dj-database-url = "^2.1.0"      # Used to configure the test database for the test app
```

Comment:
Ive since removed the need for this (as it was only really to switch back and forth between sqlite and postgres with an env var) .. but now heroku is gone, the test app need only be run locally with sqlite.

If you can remove this, and rebase this PR from `master` ill look to get it tested and merged

Reply:
Awesome, yeah I'll rebase now.

Reply:
I've rebased now, and it mostly looks good now. Apart from 3.10, the build is failing due to a missing TravisCI repo_token. Seems consistent with other PRs - https://github.com/farridav/django-jazzmin/pull/533

I'm not sure how you fixed this in your PRs as I can see some of the recent GHA build jobs passed.